privacy-policy.md

```diff
@@ -0,0 +1,73 @@
+
+Cookies: Like many websites, we use cookies and similar technologies to collect additional website usage data and to improve our Services, but we do not require cookies for many parts of our Services such as searching and looking at public user profiles. A cookie is a small data file that is transferred to your computer or mobile device. We may use both session cookies and persistent cookies to better understand how you interact with our Services, to monitor aggregate usage by our users and web traffic routing on our Services, and to customize and improve our Services. Although most web browsers automatically accept cookies, some browsers’ settings can be modified to decline cookies or alert you when a website is attempting to place a cookie on your computer. However, some Services may not function properly if you disable cookies. We honor the Do Not Track browser option to give you control over how your website visits are used to personalize your Slaent experience and ads.  (#Do Not Track settings? #)
+
+Using Our Services: We receive information when you interact with our Services, even if you have not created an account (“Log Data”). For example, when you visit our websites, sign into our Services, interact with our email notifications, use your account to authenticate to a third-party website, application, or service, or visit a third-party website, application, or service that includes a Slaent button or widget, we may receive information about you. This Log Data may include your IP address, browser type, operating system, the referring web page, pages visited, location, your mobile carrier, device information (including device and application IDs), search terms, or cookie information. We also receive Log Data when you click on, view or interact with links on our Services, including links to third-party applications, such as when you choose to install another application through Slaent. We use Log Data to make inferences, like what topics you may be interested in, and to customize the content we show you, including ads. We keep Log Data as needed for the purposes described in this Privacy Policy. 
+
+Commerce Services: You may provide your payment information, including your credit or debit card number, card expiration date, CVV code, and billing address (collectively, “Payment Information”), along with your shipping address, to complete a commerce transaction through our Services. To facilitate future purchases, we store your Payment Information (excluding CVV code) and shipping address, which you can remove from your account at any time using your account settings. We consider your Payment Information and shipping address private and do not make such information public. To facilitate order fulfillment, we collect and store information created by your purchases made through our Services (“Transaction Data”). Transaction Data may include the merchant’s name and the date, time, and amount of the transaction.
```

Comment:
Probably not relevant for a LONG time and might make sense to leave out since we can add it in once we do have it.